frontend/screens/notifications/Notifications.tsx

```diff
@@ -1,30 +1,49 @@
 import React, { Component } from 'react';
 import {
-  Platform,
   StyleSheet,
-  Text,
   View,
-  Button,
-  StatusBar
 } from 'react-native';
 import TopBar from '../../shared/TopBar';
 import BottomBar from '../../shared/BottomBar';
 import globalStyles from '../../global_styles';
+import NotificationList from './components/NotificationList';
+import { NotificationService } from '../../services/NotificationService';
+import { Subscription } from 'rxjs';
 
 interface IProps {
   navigation: any
 }
 
 export default class Notifications extends Component<IProps> {
+  private notificationSubscription: Subscription;
+  state = {
```

Comment:
Meines Wissens ist es besser ein Interface IState zu eröffnen und es oben in `Component<IProps, IState>` einzufügen.